build-system/server/new-server/transforms/modules/test/esm-env-input.html

```diff
@@ -0,0 +1,3 @@
+<head>
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
```

Comment:
Super nit but could you add a newline at the end of both files so we won't see this "⛔️" sign? :D

Reply:
Done.